pom.xml

```diff
@@ -80,7 +80,7 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <properties>
-                <scala.version>2.13.4</scala.version>
+                <scala.version>2.13.9</scala.version>
```

Comment:
maybe we merge just this change @conker84, wdyt?

Reply:
sure